modal_proto/api.proto

```diff
@@ -417,7 +417,7 @@ message FunctionGetInputsItem {
   string input_id = 1;
   FunctionInput input = 2;
   bool kill_switch = 3;
-  string function_call_id = 4;
+  reserved 4; // previously used
```

Comment:
🤯

Reply:
Unrelated, but I noticed it wasn't being used except in a test

Reply:
didn't realize you can do this in protobuf!